File: domusviewer/app/src/main/java/it/achdjian/paolo/domusviewer/setting/SettingFragment.java
package it.achdjian.paolo.domusviewer.setting;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import it.achdjian.paolo.domusviewer.R;

/**
 * Created by paolo on 14/04/16.
 */
public class SettingFragment extends PreferenceFragment{
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.preferences);
    }
}

File: temperature_monitor/app/src/main/java/it/achdjian/paolo/temperaturemonitor/settings/SettingFragment.kt
package it.achdjian.paolo.temperaturemonitor.settings

import android.os.Bundle
import android.preference.PreferenceFragment
import it.achdjian.paolo.temperaturemonitor.R

/**
 * Created by Paolo Achdjian on 7/4/17.
 */
class SettingFragment : PreferenceFragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.preferences)
    }
}